app/import/helpers/charge-merge.js

```diff
@@ -0,0 +1,65 @@
+'use strict'
+
+// TODO: this will need to use built in date
+const moment = require('moment')
```

Comment:
It has been determined that moment will not bee used when the code is moved into the system as we do no use a date library.